spec/lib/backup_restore/system_interface_spec.rb

```diff
@@ -107,12 +107,13 @@
     end
 
     context "with Sidekiq workers" do
-      before { flush_sidekiq_redis_namespace }
-      after { flush_sidekiq_redis_namespace }
+      after do
+        flush_sidekiq_redis_namespace
+      end
 
       def flush_sidekiq_redis_namespace
         Sidekiq.redis do |redis|
-          redis.scan_each { |key| Discourse.redis.del(key) }
```

Comment:
This was causing a leak in our test suite as we were deleting from the wrong namespace.